app/views/declaration/un_dangerous_goods_code.scala.html

```diff
@@ -34,7 +34,7 @@
 
 @hintHtml = {
     @messages("declaration.unDangerousGoodsCode.header.hint")
-    <a target="_blank" href=@appConfig.tradeTariffUrl>@messages("declaration.unDangerousGoodsCode.header.hint.link")</a>
+    <a target="_blank" href=@appConfig.tradeTariffUrl>@messages("declaration.unDangerousGoodsCode.header.hint.link")</a>.
```

Comment:
Is this `dot` at the end necessary? Can we move it to the messages?

Reply:
The prototype has the full-stop outside of the link.   I thought of adding it to the message but I'm sure someone would complain

Reply:
Yeah, in that case I'll pass link to the messages as argument and put this `dot` in the messages. Probably in movement frontend you'll find examples. It's better to keep it in messages. I know that this will looks weird, but it will be safe, alone dot can looks like mistake

Reply:
Good idea.   I've re-factored all of the links I've done over the last three pages.